portal/static/portal/sass/styles.scss

```diff
@@ -465,6 +465,10 @@ a[class^='button']:visited {
   background-color: $color-background-tint-quaternary;
 }
 
+hr {
```

Comment:
![hr](https://cloud.githubusercontent.com/assets/15927761/25376852/856f1eb0-299d-11e7-9eac-a226b4772ce4.png)

This is where the <hr/> element is used. Is that not OK? :P